grammar.js

```diff
@@ -969,7 +969,7 @@ module.exports = grammar({
       commaSep(field('label', $.identifier)),
     ),
 
-    argument_list: $ => seq('(', commaSep($._expression), ')'),
+    argument_list: $ => seq('(', commaSep(choice($._expression, $.compound_statement)), ')'),
```

Comment:
It would be useful to add comments in the grammar for things that are not part of the official C grammar but are here to deal partially with cpp macros.
Something like /* the compound_statement is to parse macros taking statements as arguments, e.g., MYFORLOOP(1, 10, i, { foo(i); bar(i); } */